examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -0,0 +1,160 @@
+"""
+========================================
+Release Highlights for scikit-learn 0.23
+========================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 0.23! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. For an exhaustive list of
+all the changes, please refer to the :ref:`release notes <changes_0_23>`.
```

Comment:
Is there a way to make the link to the full changelog more visible ?

Reply:
I'll put this in bold, hopefully it will stand out